sites/all/modules/custom/fj16_job_ads/fj16_job_ads.info

```diff
@@ -23,6 +24,7 @@ dependencies[] = views
 scripts[] = js/fj16_job_ads.js
 features[ctools][] = strongarm:strongarm:1
 features[ctools][] = views:views_default:3.0
+features[fe_block_boxes][] = fj16_job_ads_intro
```

Comment:
Tätä ei pitäis tulla tänne, tän kuseisen blockin sisältö pidetään tarkoituksella pois versionhallinnasta, että sen sisältö pysyy muokattavana. Eli tätä exportatessa pitää ottaa täppä pois tämän kohdalta.